modules/custom/entity_access_by_field/tests/src/Kernel/EntityAccessAccessGrantsTest.php

```diff
@@ -27,6 +27,8 @@ public function testGrantsAlter() :void {
     entity_access_by_field_node_grants_alter($altered_grants_groups12, $this->account, 'view');
     $this->assertEquals($grants, $altered_grants_groups12);
 
+    drupal_static_reset('entity_access_by_field_node_grants_alter');
```

Comment:
This needs to happen as the results of the `node_grants_alter` part this assertion is testing, we're expecting to be changed. If we don't do this we'll not see the intended changes because of the static caching.